src/redux/reducers/resereveSlice.js

```diff
@@ -4,12 +4,6 @@ import axios from 'axios';
 const baseUrl = process.env.REACT_APP_API_AUTH_URL;
```

Comment:
As discussed please address this naming issue.
```
REACT_APP_API_URL=http://localhost:3001/api/v1
REACT_APP_API_AUTH_URL=http://localhost:3001
```
so you wont need line (6). eg.  
``` 
const BASE_URL = `${process.env.REACT_APP_API_AUTH_URL}/login
```

Reply:
Oh, ok

Reply:
This woks perfectly.
![Screenshot from 2023-10-08 08-35-24](https://github.com/lincoln1883/appointment-capstone-frontend/assets/84746035/a6b10d97-e757-4ebb-a045-860c167919cf)

Reply:
Working on it. The issue is We are not getting the updated user from local storage when logging out and logging in again.